tests/net/tx_timestamp/src/main.c

```diff
@@ -277,7 +277,7 @@ static void test_timestamp_setup_all(void)
 	net_if_unregister_timestamp_cb(&timestamp_cb_3);
 }
 
-static void test_timestamp_cleanup(void)
+static void test_timestamp_cleanup(void *dummy)
```

Comment:
Why do we add this?

Reply:
the tear down function needs a parameter of void *